crates/re_viewer/src/ui/view_spatial/scene/primitives.rs

```diff
@@ -26,7 +26,9 @@ pub struct SceneSpatialPrimitives {
     pub points: PointCloudBuilder<InstancePathHash>,
 
     pub meshes: Vec<MeshSource>,
+
     pub depth_clouds: Vec<DepthCloud>,
+    pub depth_cloud_radius_boost_in_ui_points_for_outlines: f32,
```

Comment:
find it a bit odd that this one became a field just in order to pass it through to the render bridge. Could as well argue that the DrawData should have been stored here in the first place instead of `depth_clouds` plus this parameter

Reply:
yeah, it is a bit weird. It is similar to how `PointCloudBuilder` has a top-level radius-boost member though. I could clump together `depth_clouds` and `depth_cloud_radius_boost_in_ui_points_for_outlines` into one `DepthCloudBuilder` or something, but it also feels weird.

(I don't think the radius boost belongs in the `struct DepthCloud` since it is the same for all depth clouds, not an individual setting)

Reply:
I introduced a `DepthClouds`, and I think it is cleaner